### ISSUE TITLE
Wish onnx on multilabel classification

### ISSUE THREAD

**Issue by MrRace** [ThilinaRajapakse/simpletransformers#839]
I train a multilabel classification model based on  `MultiLabelClassificationModel`  and use `convert_to_onnx` to save a onnx model, and I do not do quantization. However when load the onnx model to do prediction,  the probability result of each label is totally different with the raw model's.  Intuitively the raw model and the onnx model should get the same result. 
Raw model preds:
```
raw_outputs= [[0.01879883 0.01312256 0.01422882 0.02009583 0.01698303 0.00993347
  0.83154297 0.01332855 0.01927185 0.02493286 0.02328491 0.01519012
  0.01869202 0.01081848 0.12597656 0.01049042 0.01368713 0.02488708
  0.01724243 0.0213623  0.01455688 0.03102112 0.02250671 0.02210999
  0.01919556 0.01589966 0.01864624 0.02108765 0.01898193 0.01461792
  0.09185791 0.01815796 0.0112915  0.02174377 0.01455688 0.28759766
  0.0259552  0.02452087 0.090271   0.01596069 0.02111816 0.0111618
  0.02600098 0.01530457 0.00600433 0.01390076 0.01045227 0.01282501
  0.02041626 0.01567078 0.00701141 0.01483917 0.01812744 0.02626038
  0.02319336 0.02958679 0.01525116 0.01438904 0.01602173 0.01652527
  0.01837158 0.02140808 0.12683105 0.02241516]]
```
Onnx model preds:
```
[array([[-4.141094 , -4.210304 , -2.3390095, -3.6810098, -2.6305196,
        -3.1095068, -1.3626062, -4.4024706, -4.360234 , -3.6133196,
        -3.8280015, -4.198082 , -2.7760606, -4.157262 , -4.3966017,
        -4.5244794, -4.230932 , -3.5628433, -4.1556234, -1.3415949,
        -4.461827 , -4.872239 , -2.9669342, -3.7904565, -4.11329  ,
        -4.248634 , -4.0511737, -4.4766974, -3.1279628, -4.140027 ,
        -4.7144837, -4.087776 , -4.3172994, -3.6593697, -3.582393 ,
        -4.2231693, -2.908324 , -3.8314204,  1.0453992, -4.2778587,
        -4.184877 , -4.666135 , -3.9883466, -1.873546 , -4.8218594,
        -4.001727 , -4.3879566, -4.2481623, -3.9335036, -4.005136 ,
        -4.8996396, -4.133689 , -4.549317 , -3.9497452, -3.9441628,
        -3.8419068, -3.910229 , -4.062197 , -4.4270406, -3.7248192,
        -3.926304 , -3.9367592, -4.655781 , -3.9987605]], dtype=float32)]

```
My inference on onnx model:
```
onnx_dir = "/home/data/sports/outputs/onnx/simple_roberta_www_large_ext_onnx/"
tokenizer = BertTokenizer.from_pretrained(onnx_dir)
so = rt.SessionOptions()
so.intra_op_num_threads = 1

device_type = 'cpu'
providers = ['CPUExecutionProvider'] if device_type == 'cpu' else ['GPUExecutionProvider']

onnx_path = os.path.join(onnx_dir, "onnx_model.onnx")
onnx = rt.InferenceSession(onnx_path, so, providers=providers)

model_inputs = tokenizer("my test text", return_tensors="pt")
inputs_onnx = {k: v.cpu().detach().numpy() for k, v in model_inputs.items()}
preds = onnx.run(None, inputs_onnx)
```

**Comment by MrRace** [comment]
@ThilinaRajapakse  could you help me? Thanks a lot!

**Comment by EricYangCn** [comment]
U may find answer in the following links:
https://github.com/microsoft/onnxruntime/blob/master/onnxruntime/python/tools/transformers/notebooks/PyTorch_Bert-Squad_OnnxRuntime_GPU.ipynb

**Comment by EricYangCn** [comment]
U will find answer in the following link:
https://github.com/microsoft/onnxruntime/blob/master/onnxruntime/python/tools/transformers/notebooks/PyTorch_Bert-Squad_OnnxRuntime_GPU.ipynb
